media/server/main/include/MediaPipelineServerInternal.h

```diff
@@ -120,9 +120,11 @@ class MediaPipelineServerInternal : public IMediaPipelineServerInternal, public
 
     bool renderFrame() override;
 
-    bool setVolume(double volume) override;
+    bool setVolume(double targetVolume, uint32_t duration = 0, EaseType type = EaseType::EASE_LINEAR) override;
```

Comment:
You dont need to include defaults in here, only need it in the interface (one place)